tests/test_quoteengine.py

```diff
@@ -0,0 +1,124 @@
+from datetime import datetime
+from unittest.mock import patch
+from unittest.mock import MagicMock
+
+import pytest
+
+from tululbot.utils import QuoteEngine
+
+
+class FakeResponse:
+    def get_json(self):
+        return self.json
+
+    pass
+
+
+@pytest.fixture
+def qe():
+    return QuoteEngine()
+
+
+def cdn_with_branch(branch):

```

Comment:
can you please add a verb in the function name? this is minor but a good practice nonetheless.